### ISSUE TITLE
masher.start / composer.start messages are fairly useless

### ISSUE THREAD

**Issue by AdamWill** [fedora-infra/bodhi#3360]
Since 3b655f23 , the `bodhi.master.start` fedmsg - which later became the `bodhi.composer.start` fedora-messaging message in Bodhi 4 - is pretty, well, useless.

Before that commit the message included a list of update titles under the key `updates` in the message dict. Since that commit, it includes a list of compose dicts under the key `composes`. It seems that these compose objects inside Bodhi have an `updates` attribute which is a list of update objects, but these go missing from the messages that are published. The compose dicts that go in the messages are produced by calling `compose.__json__()`, so I guess that does not include the updates for each compose in the output for whatever reason.

The compose dicts also provide nothing that make it easy from the outside understand what compose they represent. They look like this:

          {
            "security": true,
            "release_id": 28,
            "request": "stable",
            "content_type": "rpm"
          },

I don't know what the hell release has the `release_id` 28 - it is not Fedora 28, I'm pretty sure. There is probably some kind of API query you can do to figure it out, but that would require for e.g. fedmsg_meta to hit up the Bodhi API every single time it wanted to figure out in some kind of human-comprehensible way what is actually being mashed / composed here.

I'm coming at this from the angle of working on the fedmsg_meta processor, and honestly, this is the text I'm going to have it use for these messages:

"releng requested a mash of some updates"

...because that is all I can really figure out from the message content. It's not much use to anyone.

[Here's a sample message](https://apps.fedoraproject.org/datagrepper/id?id=2019-d3041ae5-2b13-4269-944e-cee4d8238c87&is_raw=true&size=extra-large) so you can see the problem.

**Comment by sebwoj** [comment]
> Before that commit the message included a list of update titles under the key updates in the message dict. 

Yes, when Bodhi creates `Compose` json object for `bodhi.composer.start` msg, `updates` field is excluded from object.

> I don't know what the hell release has the release_id 28 - it is not Fedora 28, I'm pretty sure.

`release_id` field is actually the Release id in Bodhi Database. This field does not inform user about release name(like 'F28'). Inside Bodhi backend we can fetch release from DB using `release_id` field, but outside Bodhi this infomation is useless.

We should discuss what extra fields `bodhi.composer.start` msg have to has.
Is the `long name` info(eg "Fedora 28") the only one info missing in this msg?

**Comment by AdamWill** [comment]
I mean, the obvious thing to do to match how the messages were before would be to include the updates in the message too (whether as just plain titles or IDs, or as a dict, I'm not sure). But yeah, at least including enough info to identify the release would be a start.
